spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/ColumnVectorBuilder.java

```diff
@@ -26,13 +26,6 @@
 
 class ColumnVectorBuilder {
   private boolean[] isDeleted;
-  private int[] rowIdMapping;
-
-  public ColumnVectorBuilder withDeletedRows(int[] rowIdMappingArray, boolean[] isDeletedArray) {
-    this.rowIdMapping = rowIdMappingArray;
-    this.isDeleted = isDeletedArray;
-    return this;
-  }
```

Comment:
At the time of building the vectors, `isDeleted` and `rowIdMapping` are always null, so deleting this method for now.